debuglogger/default.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package debuglogger
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type defaultEvent struct {
+	level  LogLevel
+	logger *log.Logger
+	fields []byte
+}
+
+func (e *defaultEvent) Msg(msg string) {
+	if len(msg) == 0 {
+		return
+	}
+
+	e.logger.Print("[", e.level.String(), "] ", msg, string(e.fields))
+}
+
+func (e *defaultEvent) Str(key, val string) Event {
+	e.fields = append(e.fields, ' ')
+	e.fields = append(e.fields, key...)
+	e.fields = append(e.fields, '=')
+	if strings.Contains(val, " ") {
```

Comment:
Maybe just always quote? This looks relatively expensive for a minor tweak

Reply:
I liked the simplicity of 
![image](https://user-images.githubusercontent.com/3075074/222445205-b5a0b187-7605-4cf9-91b0-76f5cafcf405.png)
where you will see `method=POST` vs `method="POST"` but we can do always.

Reply:
Done